### ISSUE TITLE
Dewikification: Migrate all existing event pages to the new events app.

### ISSUE THREAD

**Issue by lemonsaurus** [python-discord/site#386]
**Remember to target the `dewikification` branch with this pull request**

This issue must be solved _after_ #385 

Once a new system for the event pages is ready, we should migrate our existing event pages into this new system.

Please make this a separate pull request, so that we can review the system and the content separately.

**Comment by ks129** [comment]
I will work on it.

**Comment by kosayoda** [comment]
Event pages migrated.
